py.Dockerfile

```diff
@@ -14,6 +14,7 @@ ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/root/.mujoco/mujoco210/bin
 COPY . /usr/local/gym/
 WORKDIR /usr/local/gym/
 
-RUN pip install .[noatari] && pip install -r test_requirements.txt
```

Comment:
This is fine because the python foundation is not installing any more python versions and this dockerfile is only called by the CI which uses the latest version of each python, i.e. only python 3.6.15 will be used